pkg/monitor/cluster/apiservercount.go

```diff
@@ -0,0 +1,31 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var apiServerOperatorConditionsExpected = map[string]operatorv1.ConditionStatus{
+	"Available": operatorv1.ConditionTrue,
+	"Degraded":  operatorv1.ConditionFalse,
+}
+
+func (mon *Monitor) emitAPIServerCount(ctx context.Context) error {
+	ds, err := mon.cli.AppsV1().DaemonSets("openshift-apiserver").Get("apiserver", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
```

Comment:
why not emit only when NumberAvailable != DesiredNumberScheduled
```
if ds.Status.NumberAvailable != ds.Status.DesiredNumberScheduled {
		mon.emitGauge("apiserver.openshift", 1, map[string]string{
			"available": fmt.Sprintf("%d", ds.Status.NumberAvailable),
			"desired":   fmt.Sprintf("%d", ds.Status.DesiredNumberScheduled),
		})
	}
```

Reply:
Changed for now but we need to do something with those in the long run. 
https://github.com/Azure/ARO-RP/issues/803
This pattern is created scattered metrics where we get only BAD metrics... :/

Reply:
Agreed with @mjudeikis, since we're already emitting one of these per cluster already for the healthz count, it's just confusing to conditionally emit on bad metrics. Makes it really hard to tell the difference between "unknown" and "healthy" in the metrics backend.